src/backends/plugins_compat/input_plugin_compat.c

```diff
@@ -60,8 +61,28 @@ static m64p_error input_plugin_get_input(void* opaque, uint32_t* input_)
     int pak_change_requested = 0;
 
     /* first poll controller */
-    if (input.getKeys) {
-        input.getKeys(cin_compat->control_id, &keys);
+    if (!netplay_is_init())
```

Comment:
Same here, you should not modify input backends with netplay-specific code, but instead provide a new netplay input backend and make the core use it at core startup (main_run()). This new netplay input backend may rely the non-netplay input backends to get user input.